.Lib9c.Tests/Action/BuyMultipleTest.cs

```diff
@@ -363,5 +550,20 @@ public PriceData(Currency currency)
                 PriceSum = new FungibleAssetValue(currency, 0, 0);
             }
         }
+
+        public class ShopItemData
+        {
+            public ItemType ItemType { get; set; }
+
+            public Guid ItemId { get; set; }
+
+            public BigInteger Price { get; set; }
+
+            public long RequiredBlockIndex { get; set; }
+
+            public bool Buy { get; set; }
```

Comment:
What does this variable mean?

Reply:
If this variable is set to `false`, this product will be registered to the `ShopState`, but won't buy on the actual action.